nibabel/gifti/gifti.py

```diff
@@ -386,14 +385,44 @@ def metadata(self):
         return self.meta.metadata
 
 
-class GiftiImage(FileBasedImage, xml.XmlSerializable):
+class GiftiImage(xml.XmlBasedImage):
+    """
+    The Gifti spec suggests using the following suffixes to your
```

Comment:
Here, I moved the docstring from `GiftiImage.to_file_map` to `GiftiImage`, so that `to_file_map` could be moved to `XmlBasedImage`.